calculator.js

```diff
@@ -0,0 +1,82 @@
+(function() {
```

Comment:
We've learned two things: 1) classes and 2) namespaces. Since we have the basic building blocks, we'll continue building our calculator. Next thing would be to add support for memory -- check how it works on a simple calculator (MC, MR, M+, M-). Do not forget to:
1) Add a dedicated class which will be in charge for storing things, called *Memory* for example
2) Export / import it using provide / require